materialize-snowflake/sqlgen.go

```diff
@@ -304,3 +305,20 @@ FILE_FORMAT = (
 )
 COMMENT = 'Internal stage used by Estuary Flow to stage loaded & stored documents'
 ;`
+
+type tableAndFile struct {
+	Table sql.Table
+	File  string
+}
+
+// RenderTableTemplate is a simple implementation of rendering a template with a Table
```

Comment:
This comment is probably not applicable here, seems like a copy/paste from the `RenderTableTemplate` function is no longer needed for this `RenderTableAndFileTemplate` function.